presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -866,6 +866,36 @@ Examples:
 
     CALL iceberg.system.remove_orphan_files(schema => 'db', table_name => 'sample');
 
+Fast Forward Branch
+^^^^^^^^^^^^^^^^^^^
+
+This procedure advances the current branch to a more recent snapshot from another branch without replaying any intermediate snapshots.
+``branch`` can be fast-forwarded up to the ``to`` snapshot if ``branch`` is an ancestor of ``to``.
+
+The following arguments are available:
+
+===================== ========== =============== =======================================================================
+Argument Name         required   type            Description
+===================== ========== =============== =======================================================================
+``schema``            ✔️         string          Schema of the table to update
+
+``table_name``        ✔️         string          Name of the table to update
+
+``branch``                       string          The branch you want to fast-forward
+
+``to``                ✔️         string          The branch you want to fast-forward to
+===================== ========== =============== =======================================================================
+
+Examples:
+
+* fast-forward the ``dev`` branch to the latest snapshot of the ``main`` branch ::
+
+    CALL iceberg.system.fast_forward('schema_name', 'table_name', 'dev', 'main');
+
+* Here ``branch1`` branch does not exist, it will be created and fast-forwarded to the latest snapshot of the ``main`` branch ::
```

Comment:
```suggestion
* Create a new branch named ``branch1``  and set it's current snapshot equal to the latest snapshot of the ``main`` branch ::
```

Reply:
I thought it would be better to clarify in the example that ``branch1`` doesn't exist and it will be created as part of this.
How about keeping it something like this ?
```
 Here ``branch1`` branch does not exist, as part of this new branch named ``branch1`` is created and  it's snapshot would be set to the latest snapshot of the ``main`` branch ::
```

Reply:
I think grammatically, I want to remove the beginning `Here`. Your original sentence is written in passive voice - it should be active. We can keep a statement mentioning `branch1` doesn't exist yet. How about

```
* Given the branch named ``branch1`` does not exist yet, create a new branch named ``branch1``  and set it's current snapshot equal to the latest snapshot of the ``main`` branch ::
```

Reply:
I think we could remove the opening statement as it seems redundant with "create a new branch named branch1" stating it is a new branch. So maybe 

```
* Create a new branch named ``branch1`` and set its current snapshot to the latest snapshot of the main branch
```